### PR TITLE
Add unit testcase for junos_command module

### PR DESCRIPTION
**Pull request by ganeshrn**
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

junos_command

##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.3.0
```

##### SUMMARY

```
Add unit test cases for junos_command module
```

### CODE REVIEW

File: test/runner/requirements/units.txt [hunk 1]
```diff
@@ -9,3 +9,5 @@ python-memcached
 pyyaml
 redis
 unittest2 ; python_version < '2.7'
+jxmlease
+junos-eznc
```
Comment (gundalow) [inline test/runner/requirements/units.txt]: @mattclay Do we want to pull these two dependencies in for all test runs?

I see we already have `boto3` and `redis` listed,

Comment (privateip) [reply test/runner/requirements/units.txt]: Lets chat more about this... im not sure this is the best route forward.  Can we add to this weeks IRC meeting agenda please?

Comment (gundalow) [reply test/runner/requirements/units.txt]: Added to Network Agenda https://github.com/ansible/community/issues/110

**Comment by ganeshrn** [comment]
As discussed in Ansible Network meeting (11-01-2017) will in the testcase will mock import of external libraries (junos-eznc and jxmlease) in [junos.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/junos.py#L31-L44)

**Comment by ganeshrn** [comment]
@privateip @gundalow Fixed review comments and removed third party module dependency in unit test requirements. 
Found issue https://github.com/ansible/ansible/issues/20265 while adding UT, raised PR https://github.com/ansible/ansible/pull/20266 to fix it

**Comment by ganeshrn** [comment]
@privateip Removed `xml` and `json`  format related test case as behavior of these formats might change in future as per recent discussions. Added only `text` format related testcases.

**Pull request merged**